### DISCUSSION TITLE
USB Not Bootable - OC Detected in BIOS but States "No bootable device detected" - NUC8i7HVK

### DISCUSSION THREAD

**Discussion by kemjos** [osy86/HaC-Mini#581] [Q&A]
Hey everyone!

I have tried many times now to boot from the USB created by the HacMini Installer, to no avail.

My BIOS version on the NUC8i7HVK is 0066. I have tried the aforementioned using the HacMini 3.0.1 prerelease, 2.9 and 2.7 releases. I have also tried using both the Catalina 10.15.7 and Big Sur 11.3.1 installers, also without success.

I have followed [this](https://osy.gitbook.io/hac-mini-guide/) guide to the letter using 2 different Mac laptops. 

Whilst the OpenCore USB is detected as a bootable device in the BIOS settings and the Boot Menu (BIOS using F10), it simply doesnt boot and doesnt show any inforrmation at all, including using the Debug Options or OpenCore DEBUG. 

When I boot with the USB set as first boot priority, the screen turns black and then OpenCore is simply ignored and the device goes on boots on my Linux drive directly. If I disable all M.2 drives in the bios, it simple states "No bootable device found.". 

Where I'm using the Boot Menu (BIOS using F10), it shows OpenCore as an option, however when I select OpenCore and press Enter, the screen turns black and I'm brought back into the Boot Menu (BIOS using F10) where OpenCore is listed again. If I remove the USB drive, press Enter on OpenCore, the Boot Menu (BIOS using F10) still shows OpenCore as an option.

If anyone knows anything that could help me out it would be much appreciated.

**Comment by osy** [comment]
Have you selected “show boot menu” in the installer when creating the USB? That should force OC to get into the selection menu. Also if all else fails reset your bios and try again.

**Reply by kemjos** [reply]
Hi,

Thank you for responding.

Yes I have also tried with Show Boot Menu, unfortunately it doesn’t have any effect, it seems that doesn’t even reach the boot menu at all.

Resetting the BIOS has been done multiple times, the only thing I haven’t tried regarding the BIOS is to flash it anew entirely, I’ll try this instead and revert accordingly.

**Comment by Zekiee** [comment]
I met the same problem on NUC8i7HVK. BIOS 0067. WIth some custom option now it is work for me.
HacMini.pkg -> Customsize -> Boot Options -> Show boot menu (mark checked)

**Comment by kemjos** [comment]
Alright, so after abandoning the installation of MacOS due to the inability of the BIOS to boot into OpenCore, no matter the suggested options, I now realised that… OpenCore actually broke my entire system in some way.

My bios is left broken and no amount of tinkering and troubleshooting has helped. The symptoms are as follow:

- Oddly enough, every single bootable USB key plugged in the system is tagged as “OpenCore”. Some USB keys I never used on the system or new ones just out of the store, they are all tagged as OpenCore as soon as you flash ANY operating system on them.

- The BIOS cannot be recovered. Using all methods, including the security jumper and cmos battery removal, the recovery just never reaches the state of flashing the .BIO file. The system restarts and loops on the revery process beggining in X seconds.

- F10 Boot menu cannot boot any installation medium. Upon selecting whatever installation medium labeled as “OpenCore” by the faulty bios, it simply resets into the Boot Manager.

- F7 Update manager seems to flash the bios… and yet it doesn’t. It stops at a certain step and the BIOS log shows a failed bios update every time I try. I only tried to flash the 0067 bios after trying to boot from OpenCore. 

- Windows cannot be installed anymore. It shows as OpenCore, and the system just says not bootable system detected. These are fresh USB keys, with fresh Windows installs from the media creation tools.

These issues arose straight after I tried to boot into OpenCore on my BIOS on 0067. I’m now left with a system that cannot boot anything else than Linux.

The correlation between OpenCore and the appearance of these issues is undeniable due to the fact that they appeared immediately after, and that somehow, OpenCore is now burned in the system, detecting all USB key bootables as OpenCore ones.

**Reply by osy** [reply]
Reset nvram with opencore’s option.

**Reply by kemjos** [reply]
Will try that right away, thanks!

**Reply by kemjos** [reply]
Unfortunately, this is not possible so far... as mentioned in my previous post, there is no possibility for me to boot into OpenCore and access OpenCore's options.

This person seems to have the same problem as well: https://www.tonymacx86.com/threads/hades-canyon-is-no-longer-booting-from-opencore-brick.313199/

**Reply by osy** [reply]
Can you enter EFI shell?

**Reply by kemjos** [reply]
Yes I can now access the EFI shell! Could that help?

**Reply by kemjos** [reply]
I also have this error message as well, not sure if this is linked or not

[https://imgur.com/4kh4z5E](url)

**Reply by osy** [reply]
If you have OpenCore installed on a USB, you can launch it from EFI shell. Try "fs0:" or "fs1:" to change drives (type "ls" to see if it's the right drive). Once you find the right one, "cd" to EFI/OpenCore" and run "OpenCore.efi".

**Reply by osy** [reply]
Once you're in OpenCore, you can do the NVRAM reset. Then unplug the usb.

**Reply by kemjos** [reply]
When executing OpenCore, I get the following error message: 

OC: Failed to load configuration

**Reply by osy** [reply]
Okay, it seems like you didn't install OpenCore properly (did you use the HaC mini installer)? Anyways, you'll probably have better luck asking on a Hackintosh forum or the r/Hackintosh discord.

**Reply by kemjos** [reply]
I see, I did use the HaCMini installer but I’ll try again, I’ve been following the guide step by step without any issues arising. I paid special attention to details too.

I’ll try again to make sure I can get to OpenCore options.

Thank you for the help you have provided so far. In all fairness however, the Hackintosh reddit won’t be of any help in that matter, the bios itself has been corrupted is some deep way, as the most advanced recovery process for the bios is failing (removing the security jumper).

If resetting the nvram can resolve that issue then I’ll do my best trying. Otherwise I can safely say that I’ll be stuck at version 0067 and the computer has lost a great deal of its value.

**Reply by kemjos** [reply]
I tried again with version 2.7 of HaCMini and the EFI Shell now says this when executing OpenCore.efi:

Image is not an application

**Reply by osy** [reply]
Maybe RMA your device? Hac mini/opencore doesn’t touch the bios. Indeed I would be very surprised if it can considering Intel security doesn’t allow modifying bios/boot (if it’s modified you won’t be able to turn anything on at all). If you indeed are following everything correctly you probably have faulty hardware.

**Reply by kemjos** [reply]
I see, indeed therefore the hardware must have had some weakness somewhere and it may just be a coincidence that OpenCore triggered that flaw.

RMAing is impossible in the part of the world I live in, but interestingly I came upon a way to boot into the windows installer… through the EFI shell itself, which you know probably better than me haha

By manually going through the files on bootable USBs  I can launch the installers by executing their respective Bootx64.efi files.

At least I’m not stuck with Linux only now. This is far from a solution but at least the computer still can run something else that, as mentioned, Linux.

I’ll still fiddle around, see maybe if I can flash the bios via the EFI shell too. Not quite sure what I’m looking at but I’ll keep looking for a little while, maybe try to get some Intel engineer if I’m lucky.

**Reply by kemjos** [reply]
Ok so quick update, for some reason now when flashing the 2.7 version of HaCMini, not only I can get into the OpenCore menu and hit the "Reset NVRam" Option, but it automatically boots into the macOS installer. The latter which I tried so many time to do. Unfortunately, the nvram reset is to no avail. It seems that it doesn't change anything in regard to the OpenCore footprint there is in the bios whenever any bootable USB is detected. 

I'll dig in some more. If you have any idea of how I would be able to flash a .bio for say bios version 0059 or 0063 or even 0067 itself, please do let me know.

**Reply by osy** [reply]
Another possibility: you have a broken flash drive

**Reply by kemjos** [reply]
I wish, unfortunately it does that with all 5 usb keys I’ve tried and with at least 3 different OSes, Windows, vSphere ESXi and FreeDOS

**Comment by kingrhy** [comment]
this is same to me. but this issue only occurred on big sur installation media.  when I creat a big sur installation disk and patch with Hacmini3.15. I was encountered same problem.  even check "show boot manual"  but if i create catilina installation disk and patch it with same pkg with check "patch the installer".  its can boot from installation disk. but will stack somewhere.  i paste snapshots in another one discussion.
